doc/cli/npm-docs.md

```diff
@@ -8,6 +8,8 @@ npm-docs(1) -- Docs for a package in a web browser maybe
     npm home [<pkgname> [<pkgname> ...]]
     npm home .
 
+    aliases: home
```

Comment:
I agreed it basically, but it's same case with `npm tst` and I deleted it in [npm-test.md](https://github.com/watilde/npm/blob/patch/add-aliases/doc/cli/npm-test.md#synopsis). If they are same case, I will keep `npm tst` again.